Lib/test/test_memoryio.py

```diff
@@ -11,6 +11,13 @@
 import pickle
 import sys
 
+class IntLike():
```

Comment:
my bad.
BTW, do you think this should be added to PEP8, or is it obvious?
(i grepped and found only 7 places in the codebase doing this.)